src/lambda.d.ts

```diff
@@ -3,7 +3,7 @@ import {TestInterface} from "ava";
 import Docker = require('dockerode');
 
 export interface Environment {
-  [key:string]: string | null;
```

Comment:
This change feels wrong. Doesn't the environment need to be a string or null?

Reply:
The value should be anything that can be stringified.

Reply:
https://github.com/lifeomic/lambda-tools/blob/master/src/lambda.js#L20

Reply:
I can change it to match those values though.